actionpack/lib/action_controller/metal/renderers.rb

```diff
@@ -25,9 +42,16 @@ module Renderers
     # Default values are <tt>:json</tt>, <tt>:js</tt>, <tt>:xml</tt>.
     RENDERERS = Set.new
 
+    # A Hash mapping serializer names to callables.
+    # TODO: update text
+    # Default serializers are <tt>:json</tt>, <tt>:js</tt>, <tt>:xml</tt>.
+    SERIALIZERS = Hash.new.with_indifferent_access
```

Comment:
Why is it a Hash with indifferent access? We can use a normal hash here since we can define the API for how we add new serializers.